### PR TITLE
WIP adds framework for copying jekyll files to temp folder

### PR DESCRIPTION
**Pull request by trevormast**
Addresses issue #17
updates _config.yml with user values; needs authentication and ability to gather user values.

### CODE REVIEW

File: lib/clean-jekyll/user_config.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+--- 
+markdown: kramdown
+github_username: git_guy
+twitter_username: a_twit
+url: name.place.com
+baseurl: /myblog
+email: email@place.com
+description: email@place.com
+title: A Blog
```
Comment (amoose) [inline lib/clean-jekyll/user_config.yml]: Don't need this file

File: app.rb [hunk 2]
```diff
@@ -1,5 +1,38 @@
...
+  end
+end
+
+
+def commit_new_jekyll(dir, user_options)
+  # TODO create repository
+  #     create commit
+  #     push to repo!
+  #     profit!
+end
```
Comment (amoose) [inline app.rb]: Please run rubocop against this code

**Comment by trevormast** [comment]
Removed user_config.yml and ran rubocop.

**Comment by amoose** [comment]
Closing in favour of #41

**Pull request closed without merging**